revolv/static/css/screen.css

```diff
@@ -3700,6 +3702,181 @@ header.edit-project-header > .container > h1 {
+  text-align: left;
+}
+.txt-area1 .btn-i-want-to-donate {
+  background-color: #00a1d9;
+  font-size: 14px !important;
+  height: 45px;
+  letter-spacing: 4px;
+  line-height: 45px;
+  margin-top: 2.5rem;
+  padding-left: 20px;
+  padding-top: 0;
+  width: 100%;
+}
+.details-active-project-module .container {
+  padding: 4px 54px 0 78px;
+}
+/*$ 20 CSS START HERE*/
+.right_asides {
+  background: #ededed none repeat scroll 0 0;
+  float: right;
+  margin-bottom: 30px;
+  margin-top: 10px;
+  min-height: auto;
+  padding: 30px 0;
+  text-align: center;
+  width: 100%;
+}
+.outer_basic_silver_gold .container {
+  padding: 4px 55px 0 99px;
+}
+.outer_basic_silver_gold .content_gold_silver {
+  font-size: 24.19px;
+  line-height: 44px;
+}
+.right_asides aside.right-aside h1.basic {
+  color: rgb(181, 181, 181);
+  font-size: 18px;
+  font-weight: bold;
+  margin: 0 0 10px;
+  text-transform: uppercase;
+}
+.right_asides aside.right-aside strong {
+  color: rgb(40, 40, 40);
+  display: block;
+  font-size: 55px;
+  font-weight: bold;
+  padding-bottom: 15px;
+}
+.right_asides aside.right-aside span.our_apps {
+  border-bottom: 1px solid #c3c3c3;
+  border-top: 1px solid #c3c3c3;
+  color: #4b4b4b;
+  display: block;
+  font-family: "Source Sans Pro";
+  font-size: 16px;
+  font-weight: 600;
+  height: 33px;
+  line-height: 33px;
+  margin: 0 18px 0 16px;
+  text-transform: uppercase;
+}
+.right_asides aside.right-aside p.dummy {
+  color: #808080;
+  font-weight: bold;
+  line-height: 35px;
+  margin: 15px auto;
+  text-align: center;
+  width: 146px;
+}
+.right_asides aside.right-aside .btn-i-want-to-donate {
+  background-color: #00a1d9;
+  font-size: 14px !important;
+  height: 45px;
+  letter-spacing: 4px;
+  line-height: 45px;
+  margin-top: 2.5rem;
+  padding-left: 20px;
+  padding-top: 0;
+  width: 75%;
+}
+.txt-area1 .module-box .input-group-addon {
+	border-radius: 0px !important;
+	padding: 6px 15px !important;
+}
+.module-box .input-group .form-control {
+  border-radius: 0 !important;
+  float: left;
+  height: 40px !important;
+  margin-bottom: 0;
+  position: relative;
+  width: 100%;
+  z-index: 2;
+}
+p.content_sec {
+  font-size: 16px !important;
+  line-height: 37px !important;
+  margin-top: 40px;
+}
+/*$ 20 CLOSE HERE*/
+
+/*close here*/
+/*30th of june css begin here*/
+
+.outer_sil_gold {
+  padding-left: 0 !important;
+  padding-right: 0 !important;
+}
+.main-area1 .container,
+.outer_tab .container,
+.bg_inner {
+  padding: 4px 65px 0 99px;
+}
+.bg_inner {
+  padding: 0px 65px 0 99px;
+}
+.main-area1 .venue-area h2 {
+  margin-top: 40px;
+  padding: 0 28px;
+}
+.donors-module .mains-tabs {
+  margin-top: 40px;
+}
+li:before.active > *::before {
+  background: rgb(0,161,216);
+  content: "";
+  display: block;
+  height: 4px;
+  top: 23px;
+  width: 100%;
+}
+.donors-module .row .col-md-6 {
+  padding: 46px 0 0;
+}
+.donors-module .info-box {
+  margin-left: 128px;
+}
+/*closed here 30th june css*/
```

Comment:
This comment is not necessary, please remove it. If you like, you can leave another comment here saying that this is the end of the section that the new styles for the feature apply to.